### PR TITLE
Add exposing-a-restful-resource-using-the-http-connector sample

### PR DESCRIPTION
**Pull request by rosensilva**

### CODE REVIEW

File: migration/mule/exposing-a-restful-resource-using-the-http-connector/ExposeRESTfulService/src/main/synapse-config/api/RESTfulService.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            <respond/>
+        </inSequence>
+        <outSequence/>
+        <faultSequence/>
+    </resource>
+    <resource methods="GET" uri-template="/{personId}">
+        <inSequence>
+            <payloadFactory description="Mock Payload Retrieve one " media-type="json">
+                <format>{
+   "personId":"1",
```
Comment (abeykoon) [inline migration/mule/exposing-a-restful-resource-using-the-http-connector/ExposeRESTfulService/src/main/synapse-config/api/RESTfulService.xml]: Rather returning "1" as `personId`, shall we return with path parameter customer is using to query?

File: migration/mule/exposing-a-restful-resource-using-the-http-connector/README.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+	{ "personId": "1", "message": "Person was created"}
+
+Finally, WSO2 EI sends the message back to the client using [respond mediator](https://ei.docs.wso2.com/en/latest/micro-integrator/references/mediators/respond-Mediator/).
+
+### GET /person/{personId} Resource (2nd Resource)
+
+This resource is responsible for retrieving a specific person's data, based on the provided person ID. It starts with a WSO2 API Resource, which listens at http://localhost:8290/person/{personId}.
+
+Next, the flow will create a mock person detail using the PayloadFactory mediator and send the response using Respond mediator.
```
Comment (abeykoon) [inline migration/mule/exposing-a-restful-resource-using-the-http-connector/README.md:86]: Should we add a note here stating, that we are retuning a static payload?

File: migration/mule/exposing-a-restful-resource-using-the-http-connector/ExposeRESTfulService/src/main/synapse-config/api/RESTfulService.xml (hunk 1 above)
Comment (rosensilva) [reply migration/mule/exposing-a-restful-resource-using-the-http-connector/ExposeRESTfulService/src/main/synapse-config/api/RESTfulService.xml]: Fixed

File: migration/mule/exposing-a-restful-resource-using-the-http-connector/README.md (hunk 2 above)
Comment (rosensilva) [reply migration/mule/exposing-a-restful-resource-using-the-http-connector/README.md:86]: Updated

**Review by abeykoon — APPROVED** [review]

**Pull request merged**